### PR TITLE
Update Prow setup document

### PR DESCRIPTION
**Pull request by adrcunha**
* remove irrelevant sections
* update section about Boskos
* add section about setting up the issue tracker

### CODE REVIEW

File: ci/prow_setup.md [hunk 1]
```diff
@@ -1,62 +1,28 @@
...
 
 ## Expanding Boskos pool
 
 1. All projects and permissions can be created by running
-   `./ci/prow/boskos/create_projects.sh`. For example, to create 10 projects
-   named `knative-boskos-51`, `knative-boskos-52`, ... `knative-boskos-60`, run:
-   `./ci/prow/boskos/create_projects 51 10 0X0X0X-0X0X0X-0X0X0X /tmp/successful.out`.
-   You will need to substitute the actual billing ID for the third argument. In
+   `./ci/prow/create_boskos_projects.sh`. For example, to create 10 extra projects
+   run `./ci/prow/boskos/create_projects 10 0X0X0X-0X0X0X-0X0X0X`.
```
Comment (adrcunha) [inline ci/prow_setup.md]:
```suggestion
   run `./ci/prow/create_boskos_projects 10 0X0X0X-0X0X0X-0X0X0X`.
```

File: ci/prow_setup.md [hunk 2]
```diff
@@ -146,3 +108,11 @@
 1. Set the new test jobs as required status checks for the master branch.
 
    ![Branch Checks](branch_checks.png)
+
+## Setting up the issue tracker for a new repo
+
+1. Update [`generateIssueTrackerPeriodicJobs()`](https://github.com/knative/test-infra/blob/585becc692106854a4de992c4dda9106addddb8e/ci/prow/issue_tracker_config.go#L48)
+   in (issue_tracker_config.go)[./prow/issue_tracker_config.go), adding the call
```
Comment (adrcunha) [inline ci/prow_setup.md]:
```suggestion
   in (issue_tracker_config.go)[./prow/issue_tracker_config.go], adding the call
```

File: ci/prow_setup.md [hunk 3]
```diff
@@ -146,3 +108,11 @@
 1. Set the new test jobs as required status checks for the master branch.
 
    ![Branch Checks](branch_checks.png)
+
+## Setting up the issue tracker for a new repo
+
+1. Update [`generateIssueTrackerPeriodicJobs()`](https://github.com/knative/test-infra/blob/585becc692106854a4de992c4dda9106addddb8e/ci/prow/issue_tracker_config.go#L48)
+   in (issue_tracker_config.go)[./prow/issue_tracker_config.go], adding the call
```
Comment (adrcunha) [inline ci/prow_setup.md]:
```suggestion
   in [issue_tracker_config.go](./prow/issue_tracker_config.go), adding the call
```

File: ci/prow_setup.md [hunk 4]
```diff
@@ -146,3 +108,11 @@
 1. Set the new test jobs as required status checks for the master branch.
 
    ![Branch Checks](branch_checks.png)
+
+## Setting up the issue tracker for a new repo
+
+1. Update [`generateIssueTrackerPeriodicJobs()`](https://github.com/knative/test-infra/blob/51c37921d4a7722855fcbb020db3c3865db1cb8f/ci/prow/issue_tracker_config.go#L48)
+   in [issue_tracker_config.go](./prow/issue_tracker_config.go), adding the call
+   to generate the config for the new repo.
```
Comment (chaodaiG) [inline ci/prow_setup.md]: should we also add `make config` step here?

Comment (adrcunha) [reply ci/prow_setup.md]: Indeed. Actually there was a bunch of outdated instructions regarding the config, and I updated them too.

**Pull request merged**